pkg/simple/client/elasticsearch/esclient.go

```diff
@@ -476,24 +478,24 @@ func parseQueryResult(operation int, param QueryParameters, body []byte, query [
 	case OperationStatistics:
 		var statisticsResponse StatisticsResponseAggregations
 		err := jsonIter.Unmarshal(response.Aggregations, &statisticsResponse)
-		if err != nil {
+		if err != nil && response.Aggregations != nil {
```

Comment:
Note: response with null aggregation from server should not be treated as server error.